encodings/caesar/caesar.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import caesar_args
+
+# All input should be uppercased
+CHARACTER_OFFSET = 65
+ALPHABET_SIZE = 26
+
+def shift_letter(letter, shift_count):
+    char_code = ord(letter) - CHARACTER_OFFSET
+    if char_code < 0:
+        print("Error, invalid letter: " + letter)
+    else:
+        char_code = (char_code + shift_count) % ALPHABET_SIZE
+    return chr(char_code + CHARACTER_OFFSET)
+
+def shift_word(word, shift_count):
+    new_word = ''
+    for letter in word:
+        new_word += shift_letter(letter, shift_count)
+    print(new_word)
+    return
```

Comment:
Similarly, I would return the result here and have the caller decide whether to print it or use it for something else. Same goes for some of the other functions that prints the result instead of returning.